src/lib.rs

```diff
@@ -21,8 +21,9 @@ use std::{
 };
 use tokio::runtime::Runtime;
 use up_rust::{ComparableListener, UAttributes, UCode, UListener, UPriority, UStatus, UUri};
+// Re-export Zenoh config
+pub use zenoh::config::Config;
```

Comment:
Is this enough to configure a Config completely? Or should we re-export all of zenoh::config::*?

Reply:
It seems like there will be some conflicts if expose all config. I'll deal with it later.